context/app/static/js/components/workspaces/types.ts

```diff
@@ -145,12 +145,22 @@ interface CreateTemplateNotebooksTypes {
   workspaceName: string;
   workspaceJobTypeId: string;
   workspaceResourceOptions: WorkspaceResourceOptions;
+  onCreateWorkspace?: ({ name, files, symlinks }: { name: string; files: string[]; symlinks: string[] }) => void;
 }
 
 type TemplateTags = Record<string, string>;
 
 type TemplateTagsResponse = WorkspaceAPIResponse<TemplateTags>;
 
+/* eslint-disable-next-line no-shadow */
```

Comment:
I'm curious - what is this ESLint rule is picking up as a shadow declaration?

Reply:
Great question - haven't been able to figure this one out! The additional declaration is cited to the same line as this one, and if the name of the enum is changed the error persists with the updated name.

Reply:
Resolved by replacing `no-shadow` rule with `'@typescript-eslint/no-shadow'` rule in `.ts` files.